tests/readme-icons.test.js

```diff
@@ -0,0 +1,34 @@
+import { test, exec } from 'uvu';
+import * as assert from 'uvu/assert';
+import path from 'path';
+import * as fs from 'fs';
+import { fileURLToPath } from 'url';
+
+(async () => {
+  test(`Check readme light and dark icons`, () => {
+    const __dirname = path.dirname(fileURLToPath(import.meta.url));
+    const lightIconsPath = path.resolve(__dirname, '../assets/readme');
+    const darkIconsPath = path.resolve(__dirname, '../assets/readme');
```

Comment:
These path-related variables can be placed outside or they will be executed multiple times.

Reply:
fixed the path.
The PR is salvageable as is, no need to revert.